docs/core-concepts/working-with-templates/data-objects.md

```diff
@@ -0,0 +1,47 @@
+# Data Available in Templates
+
+Apostrophe makes different data available for you in templates depending on the context. Most of this is covered in other sections as well, but this page serves to collect that information in one place, as well as highlight options not covered elsewhere.
+
+::: tip
+Anything attached to the `req.data` object becomes visible as the `data` object in Nunjucks templates when rendering pages and other templates. Before a page is rendered, you can add properties onto `req.data` to make any new information that you need available in a template.
+:::
+
+## `data.global`
+
+`data.global` is available everywhere and always represents the same `apostrophe-global` document with the same information everywhere. Read more about this on the [global doc reference page](/core-concepts/global-settings/global.html#the-global-doc-sharing-content-across-pages).
+
+## `data.page`
+
+The `data.page` object is available in any page, piece "show page" (e.g., `show.html` -- these aren't "pages" in the sense that they might extend `apostrophe-pages`), and widget template. It contains information about the current page's document (e.g., title, areas). There is a bit more to think about if  you are using it on show page or widget templates, but most of the time you will probably be using this directly on a normal page or index page template.
+
+In `show.html` templates it references the index page that the show "page" belongs to at a given URL (if you have one index page for the piece type this is simple, with multiple index pages for a type it's slightly more complicated).
+
+In widget templates, `data.page` references the page document that the widget is on. If the widget is specifically tied to a "piece," see above.
+
+## `data.piece`
+
+This is available in show page template for a piece as well as any widgets that belong to a piece document. Similar to `data.page`, it contains information about a piece and its properties.
+
+## `data.widget`
+
+Only available in a widget template! This object includes information about that widget and its properties. It's not its own document in the database, but is a small piece of a document.
+
+## `data.user`
+
+The currently active user. This is especially useful as a quick way to see if the active user is logged out (it will be `undefined`) or not.
+
+## `data.query`
+
+This is used less, but can be very useful. `data.query` contains the contents of `req.query`. So if someone reaches the page at a url with query parameters such as `https://example.com?color=blue`, `data.query` would be `{ color: 'blue' }` in that the page template.
+
+## Other properties
```

Comment:
Wondering if we should also mention `data.when` here.

Reply:
Probably. I'm nervous about trying to encapsulate what scenes are, given they aren't explained succinctly elsewhere.